### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by EidHachem**
## In this PR I implemented the below:

- Used `TSDEV-REACT-DONUT-CHART` dependency to simulate the donut chart.
- Added proper styling to match the template from `zeplin.io`.

Many thanks for considering my request.
Happy coding 🎉🎉🎉

### CODE REVIEW

File: src/Styles/App.module.css [hunk 1]
```diff
@@ -1,16 +1,21 @@
 .navbar {
```
Comment (Ismailco) [inline src/Styles/App.module.css:1]: - [x] The nav bar has a profile icon, kindly add it to your design.  you should replicate the Zeplin design

|Your design|Zeplin design|
|-|-|
|Screenshot_20220609_193055|Screenshot_20220609_193441|

File: src/Styles/Form.module.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .form {
```
Comment (Ismailco) [inline src/Styles/Form.module.css:1]: - [x] Try to duplicate the Zeplin design, it should be identical to your app, kindly use the same padding/margin, font size, and color

|Your design|Zeplin design|
|-|-|
|Screenshot_20220609_193041|Screenshot_20220609_193901|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @EidHachem,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @EidHachem,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**